docs/release_notes/may_2024.md

```diff
@@ -0,0 +1,28 @@
+# May 2024
+
+## airbyte v0.58.1 to v0.62.2
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## ✨ Highlights
+
+Our certified database sources (Mongo,  MySQL, MSSQL and Postgres) are now [resilient](/operator-guides/refreshes#resumability) to failures while syncing with full refresh sync mode. They’ll now checkpoint so that we resume syncing between attempts without needing to start over.
+
+We expanded on our AI offering by releasing a new [Snowflake Cortex](/integrations/destinations/snowflake-cortex) destination to support users who prefer to use Snowflake as a vector store and utilize the LLM capabilities of Snowflake Cortex functions.
+
+
+## Platform Releases
+
+- We are now resilient to changes when a connector changes its defined primary key. This can occur during major version updates to a connector or an upstream adjustments to a source-defined primary key. Airbyte will gracefully recognize the change in the primary key and ask to update the connection before syncing again.
+
+- (Self-Managed Enterprise only) Enterprise configurations are now more secure and easier to support by deprecating the airbyte.yaml file as the standard way of configuring Enterprise deployments. All configuration is now sources from user-managed secrets.
```

Comment:
I'm not sure I'm reading this correctly. Would it be clearer (while still accurate) to say:  "We've made Enterprise configurations more secure and easier to support by deprecating the use of the airbyte.yml file. Now, all Enterprise configuration is sourced from user-managed secrets." ?

Reply:
That's much better than what I had! Adjusted to match your wording.